### DISCUSSION TITLE
Passing arguments to setup.exe

### DISCUSSION THREAD

**Discussion by CrowMash** [tauri-apps/tauri#6942] [Q&A]
Hi!

Is it possible to pass arguments while executing the setup.exe on Windows? I'd like to be able to deploy my .exe on several computers but run them with different parameters.

Thanks!

**Comment by simonhyll** [comment]
You should be able to extend the NSIS script used fairly easily (I haven't done so yet so not 100% sure how, I just know you can). The NSIS script allows you to add arguments, or even entire extra setup pages if you like, it's one of the nice things about switching from Wix, it's so easy to extend NSIS
